proxy/core/src/main/scala/io/cloudstate/proxy/AkkaSerializerRegisterFeature.scala

```diff
@@ -11,23 +11,58 @@ import java.util.concurrent.ConcurrentHashMap
 @AutomaticFeature
 final class AkkaSerializerRegisterFeature extends Feature {
   private[this] final val cache = ConcurrentHashMap.newKeySet[String]
-  override final def duringAnalysis(access: Feature.DuringAnalysisAccess): Unit = {
-    val akkaSerializerClass =
-      access.findClassByName(classOf[akka.serialization.Serializer].getName) // We do this to get compile-time safety of the classes, and allow graalvm to resolve their names
+  private[this] final val registerTheseSerializers = {
+    val c = ConcurrentHashMap.newKeySet[String]
+    c.add("akka.serialization.NullSerializer$")
+    c
+  }
+
+  override final def beforeAnalysis(access: Feature.BeforeAnalysisAccess): Unit = {
+    val config = com.typesafe.config.ConfigFactory.load()
+    config
+      .getConfig("akka.actor.serializers")
+      .root
+      .unwrapped
+      .values
+      .iterator
+      .asScala
+      .map(_.toString)
+      .foreach(registerTheseSerializers.add)
+    com.typesafe.config.ConfigFactory.invalidateCaches()
```

Comment:
This is an attempt to make sure that we don't lock these values into place at build time, rather than at runtime.